src/Video.js

```diff
@@ -840,6 +820,20 @@ export default class Video extends React.Component {
     this.bufferingOpacity?.setValue(0);
   };
 
+  onBuffer = ({ isBuffering }) => {
+    clearTimeout(this.bufferingTO);
+    clearTimeout(this.bufferingTooLongTO);
+    if (!aCasting && !gCasting && !this.props.youtubeId) {
+      this.bufferingTO = setTimeout(
+        () => this.bufferingOpacity.setValue(this.state.pasued ? 0 : isBuffering), 3000
```

Comment:
@csillaj spelling mistake here. "pasued". 

I would change this, but I don't have access to push to this package. I changed this locally for testing, and it was working :)